osu.Game/Screens/Ranking/Statistics/HitEventTimingDistributionGraph.cs

```diff
@@ -79,105 +104,174 @@ private void load()
+                        RelativePositionAxes = Axes.X,
+                        X = position / 2,
+                        Alpha = alpha,
+                        Text = axisValue.ToString("+0"),
+                        Font = OsuFont.GetFont(size: axis_font_size, weight: FontWeight.SemiBold)
+                    });
+                }
             }
         }
 
         private class Bar : CompositeDrawable
         {
-            public Bar()
+            private readonly float value;
+            private readonly float maxValue;
+
+            private readonly Circle boxOriginal;
+            private Circle boxAdjustment;
+
+            private const float minimum_height = 0.05f;
+
+            public Bar(float value, float maxValue, bool isCentre)
             {
-                Anchor = Anchor.BottomCentre;
-                Origin = Anchor.BottomCentre;
+                this.value = value;
+                this.maxValue = maxValue;
 
                 RelativeSizeAxes = Axes.Both;
+                Masking = true;
 
-                InternalChild = new Circle
+                InternalChildren = new Drawable[]
                 {
-                    RelativeSizeAxes = Axes.Both,
-                    Colour = Color4Extensions.FromHex("#66FFCC")
+                    boxOriginal = new Circle
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Anchor = Anchor.BottomCentre,
+                        Origin = Anchor.BottomCentre,
+                        Colour = isCentre ? Color4.White : Color4Extensions.FromHex("#66FFCC"),
+                        Height = minimum_height,
+                    },
                 };
             }
+
+            private const double duration = 300;
+
+            protected override void LoadComplete()
+            {
+                base.LoadComplete();
+
+                float height = Math.Clamp(value / maxValue, minimum_height, 1);
+
+                if (height > minimum_height)
+                    boxOriginal.ResizeHeightTo(height, duration, Easing.OutQuint);
+            }
+
+            public void UpdateOffset(float adjustment)
+            {
+                bool hasAdjustment = adjustment != value;
+
+                if (boxAdjustment == null)
+                {
+                    if (!hasAdjustment)
+                        return;
+
+                    AddInternal(boxAdjustment = new Circle
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Anchor = Anchor.BottomCentre,
+                        Origin = Anchor.BottomCentre,
+                        Colour = Color4.Yellow,
+                        Blending = BlendingParameters.Additive,
+                        Alpha = 0.6f,
+                        Height = 0,
+                    });
+                }
+
+                boxAdjustment.ResizeHeightTo(Math.Clamp(adjustment / maxValue, 0.05f, 1), duration, Easing.OutQuint);
```

Comment:
```suggestion
                boxAdjustment.ResizeHeightTo(Math.Clamp(adjustment / maxValue, minimum_height, 1), duration, Easing.OutQuint);
```

is what I'm guessing the intent here was

Reply:
Yeah, not really sure how this happened. I'm 99% sure it was using the `const` when I wrote it..